### ISSUE TITLE
Dealing with targeted comment downvotes

### ISSUE THREAD

**Issue by Voileexperiments** [Codewars/codewars.com#1438]
It's a well known fact that power user JohanWiltink has been getting targeted comment downvotes for *months*. Well, guess what, some power users (including me) has got these targeted downvotes to all the recent comments too recently, presumably because someone doesn't like the tone of the comments.

Given that downvotes:
- reduce a user's honor
- has effect on other user's impression on the comment
- is an abuse of the feature (see: stackoverflow)

, shouldn't this be detected and handled, and presumably punish the offenders?

**Comment by Voileexperiments** [comment]
Does it have to do with people abusing the comment voting system? :wink: If not, why are you leaving a comment here when the comment has nothing to do with the issue itself? :wink: :wink:

**Comment by DonaldKellett** [comment]
@AnvShx While certain "wise-ass" comments may deserve to be downvoted, that doesn't mean that *all* comments of that particular user deserve to be downvoted regardless of context so your point is moot.

BTW two things came to mind:

1. Ad-hominem attacks are a sure sign that the attacker is losing an argument and therefore is trying to redirect attention away from the actual problem at hand in a desperate attempt to save face :wink: 
2. The manner in which you commented on this Issue Report makes me wonder whether you're one of them doing the targeted downvoting ... @kazk would you mind checking on this user (if he/she linked their CW account to their GitHub account) to see if that's the case?  Cheers 😄

**Comment by AnvShx** [comment]
@DonaldKellett A few things also came to my mind -
1. The problem of targeted downvoting, in my opinion, is a secondary one. The larger problem is the attitude of people who choose to berate/insult people asking for help on kata discourse with comments like "**If you don't know how to solve this kata, maybe you shouldn't be here**". A person who is trying to learn something new, and trying to solve a problem by himself and asking for help shouldn't be berated about his lack of knowledge, just because other "power users" consider this right. If only said power users took a moment of their time to actually help people with a simple hint on how to solve a kata, rather than taunting them, things (targeted downvoting) wouldn't be as bad as they are currently.
2. As for your suspicions on whether I'm one of those doing the targeted downvoting, I could really care less about it. If I can help, I'd rather give a simple hint to the user asking for help and move on to my next kata, than waste my time downvoting a "wise-ass" comment, much less go looking for all such comments made by the user and start downvoting all of them. So, by all means, go ahead and check all you want. The fact that you chose to consider whether I'm one of those doing the targeted downvoting, instead of looking at the larger underlying problem, tells me that this issue will never be solved.

Cheers.

**Comment by GiacomoSorbi** [comment]
As much as @Voileexperiments might be a pain in the ass or not (I have never had any serious issues with him, but I understand it might hit hard on some beginners), I think we are discussing something a bit less *ad personam* and a bit more general; particularly on the lower end of the ranking, being harassed by some kind of vengeful stalker is a big let down.

Now, tackling the issue by identifying the unusual patterns should not be a major issue and would improve significantly the overall CW experience.

Can we agree on this, without going personal?

**Comment by Voileexperiments** [comment]
Yeah, no matter whether you don't like someone or not, it's never any grounds to downvote *everything that people has posted*; it's clearly an abuse of the system, which is specifically about the comments in question, not the person in question. In a sense this protects *both sides* so that we don't descend into botnet combat. If someone is overly toxic (which I'm not, btw :wink:) or are clearly spamming, you should *report to the site admin*, which is why I'm raising an issue about this here instead of just doing the same to set it back myself.

**Comment by AnvShx** [comment]
@Voileexperiments You're not overly toxic?

> ES6 is the new JS standard. **If you're still staying with ES5 like Codecademy and FreeCodeCamp for whatever arcane reasons and blame people for using quality of life ES6 features, I'd say you have a bigger problem than the one who write ES6 code.**
Also, check your usage on "clever". It does not mean "code that I don't like".

> "constructive engagement"? **Ha, funny that you're saying that in the first place with that kind of comment to begin with.**
> A truly constructive engagement would be

> > This solution uses destructuring, which is a ES6 feature. However, some browsers, mostly IE (link to "can I use" page) does not support this. In ES5 this would be blablabla...

> Instead you're just charging in with a "let's bitch about solutions I don't like without explaining why! **And in the meantime, let's put more unsolicited jabs towards short, smart code that I don't like!" comment, and then accuse others of "personal attack" when being called out. Even the arguments you use are the same bogus cliches.**

> **People like you occur in CW like once every week, so literally every power user know that this is bad, your comment is bad and you should feel bad. You're not even trying to be informative. And if you're here to learn, your attitude is the contrary of what you're saying you're doing, really** ;-)

> Also, for the record, CW has removed Node 0.10 version recently, **so you can hold whatever ES5 superiority you have, but the truth is ES6 is going to stay with us**.

> Can we have any people that complains about solutions with consistent reasons that actually makes sense? **Because as far as I can see it's almost always the opposite kind of people who barges in and begin complaining about everything, then forcefully try to get away with it by keep saying "I will not continue this discussion any further with all the hostility towards me". If you want to masturbate like that, go somewehre else.**

> If some solutions fail against some test cases, then they should not pass. If you're still allowing them to pass, it's an issue, **the kata is bad and you should feel bad**. You might as well write no tests, since you're now literally saying correctness is not your concern. Tests are what enforces and validates your specs, and it's your responsibility as a kata author to write thorough and complete tests.

It looks like making others **feel bad about themselves** is quite a specialty of yours, no? As you clearly instructed me [earlier](https://github.com/Codewars/codewars.com/issues/1438#issuecomment-390501867) -

> If you're going to put a blind eye to that, you're biased and you should feel bad.

It's funny, I had to go back only a few days to see this kind of gold on the discourse. I can't even begin to wonder how nice you've been to others throughout your time as a power user.

Oh yeah, since you asked -

>  If you're picking a fight you don't have solid grounds on, don't expect to get away with it. I'm such kind of person.

I know now exactly what kind of a person you are. I know I have pretty solid grounds too. 

It's just a pity reporting ***you*** to the site admin would do no good, so I might as well drop it here. I'm here just to learn and practice, not waste my time on people like you.

Cheers.

**Comment by GiacomoSorbi** [comment]
I am not taking sides, but I am not even sure why a site admin should be included about @Voileexperiments ' behaviour, not breaking any law or rule or whatevs; that said, can we keep personal matters outside, as previously stated? Or, worst case, open a new issue demanding a change or stronger enforcement of the code of conduct (do we even have one? Not sure, but be my guest).

**Comment by Voileexperiments** [comment]
Since he's obviously not going to actually do anything contributive (just look at his behaviour in this thread!), I've opened one myself.

#1440
